dev-vcs/tortoisehg/Manifest

```diff
@@ -1,3 +1 @@
-DIST tortoisehg-4.9.1.tar.gz 7960667 BLAKE2B f8a6ed65967c6389b65fd9388b3e898bf4ce09e7f95dd88467a10c061b9454eab76d0308905fd417575bf6d75dd7129a2f303bc774fd8e68824b33c18fd8054c SHA512 aef1b0d6047114794bfe78de2c0def83748e238917ce44fb56d562d53c26666dbb385867da70c4625893b148e2affa3766e40292348b78aff428df5a77d8f902
```

Comment:
Do not remove old versions and add new in the same commit.

Reply:
Why?